prow/config.yaml

```diff
@@ -105,6 +105,17 @@ branch-protection:
                 - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2"
                 - "ci/circleci: e2e-pilot-cloudfoundry-v1alpha3-v2"
                 - "merges-blocked-needs-admin"
+            authz-v2:
+              protect: true
```

Comment:
We don't need to make it protect, I think only admin is allowed to merge if it's protected.

Reply:
Deleted. Thanks Yangmin!